### PR TITLE
Add typings for pkijs, asn1js and pvutils

### PR DESCRIPTION
**Pull request by microshine**
case 1. Add a new type definition.
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

### CODE REVIEW

**Comment by rmhrisk** [comment]
Any chance these could be merged?

**Comment by vvakame** [comment]
*asn1js/index.d.ts*

Checklist

* [ ] is correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/asn1js
  * http://bower.io/search/?q=asn1js
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (asn1js-tests.ts or asn1js-tests.tsx)
* [ ] pass the Travis CI test?

------

*pkijs/index.d.ts*

Checklist

* [ ] is correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/pkijs
  * http://bower.io/search/?q=pkijs
  * others?
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (pkijs-tests.ts or pkijs-tests.tsx)
* [ ] pass the Travis CI test?

------

*pvutils/index.d.ts*

Checklist

* [X] is correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)?
  * https://www.npmjs.com/package/pvutils - https://github.com/PeculiarVentures/pvutils#readme
* [X] has a [test file](http://definitelytyped.org/guides/contributing.html#tests)? (pvutils-tests.ts or pvutils-tests.tsx)
* [ ] pass the Travis CI test?

File: asn1js/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,672 @@
+// Type definitions for asn1js
+// Project: https://github.com/PeculiarVentures/ASN1.js
+// Definitions by: Stepan Miroshin <https://github.com/microshine>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+declare module "asn1js" {
```
Comment (vvakame) [inline asn1js/index.d.ts]: could you strip `declare module` ?
https://github.com/Microsoft/types-publisher#fields-in-datadefinitionsjson
> ProperModule: Only top-level import and export declarations. Preferred

**Comment by microshine** [comment]
@vvakame I updated files. Any changes these could be merged?

**Pull request merged**